controllers/TodoControl.js

```diff
@@ -0,0 +1,75 @@
+const { Todo } = require('../models')
+
+class TodoControl {
+    static show(req, res) {
+        Todo.findAll()
+            .then(data => res.status(200).json(data))
+            .catch(err => res.status(500).json(err))
+    }
+
+    static create (req, res) {
+        let obj = {
```

Comment:
Lebih ke penamaan aja sebenernya, karena data yang mau dimasukkan adalah data `todo` lebih baik nama variablenya diganti dengan `todo`

```suggestion
        let todo = {
```

Reply:
oke kak